Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ruby:2.5
+
+MAINTAINER eric.kascic@stelligent.com
+
+RUN gem install cfn-nag
+
+ENTRYPOINT ["cfn_nag"]
```

Comment:
maybe we need one for scan v. regular?

Reply:
Not sure I follow

Reply:
there are two command lines: cfn_nag and cfn_nag_scan.  the latter will search recursively through a directory structure for cfn templates and aggregate the results

Reply:
We should be able to use both with the same Dockerfile, just default to one or the other, no?